docs/content/configuration.md

```diff
@@ -347,6 +347,65 @@ services:
     url: https://s2/
 ```
 
+#### GCP Metadata Token
+
+OPA will authenticate with a GCP [access token]() or [identity token](https://cloud.google.com/run/docs/securing/service-identity) fetched from the [Compute Metadata Server](https://cloud.google.com/compute/docs/storing-retrieving-metadata). When one or more `scopes` is provided an access token is fetched. When a non-empty `audience` is provided an identity token is fetched. An audience or `scopes` array is required.
```

Comment:
🤔 Are we missing a link for `[access token]()` or is there some markdown magic going on without me noticing...?

Reply:
Added missing link.